readme.md

```diff
@@ -100,20 +100,47 @@ The following guides are available to help you get started with the Toolkit:
 * [Use the Toolkit with Angular](https://learn.microsoft.com/graph/toolkit/get-started/use-toolkit-with-angular)
 * [Build a productivity hub app](https://learn.microsoft.com/en-us/graph/toolkit/get-started/building-one-productivity-hub)
 
-You can use the components by referencing the loader directly (via unpkg), or installing the npm package
+You can use the components by installing the npm packages or referencing the loader directly (via unpkg).
```

Comment:
Just to confirm this will get changed in the other PR, right?

Reply:
If you mean the mgt-loader removal, then yes.